tests/pkg_hacl/Makefile

```diff
@@ -0,0 +1,13 @@
+include ../Makefile.tests_common
+
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno \
+                   jiminy-mega256rfr2 mega-xplained waspmote-pro \
+                   chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+
+TEST_ON_CI_WHITELIST += native
```

Comment:
The `make test` command takes `9` seconds (including make term and reset) on `samr21-xpro` so could be activated on `ci`.
It can also be noticed as there is no `timeout` configured in the python test.

Reply:
Ack, thanks for testing

Reply:
Changed to `all`